config/api.yml

```diff
@@ -848,6 +848,8 @@
       - :name: read
         :identifier: cloud_volume_snapshot_view
       :post:
+      - :name: revert
+        :identifier: cloud_volume_snapshot_revert # doesn't exist yet
```

Comment:
I don't see any cloud providers implementing snapshot revert, @blomquisg correct me if I'm wrong but it might be because they actually snapshot cloud volumes not instances.  At least in GCE you can't revert to a snapshot volume, you just create a new instance from the snapshot, not sure if that's the same in AWS/Azure/etc...